app/screens/channel/channel_nav_bar/channel_title/index.js

```diff
@@ -30,7 +30,7 @@ function mapStateToProps(state) {
         currentChannelName: currentChannel ? currentChannel.display_name : '',
         isArchived: currentChannel ? currentChannel.delete_at !== 0 : false,
         displayName: state.views.channel.displayName,
-        channelType: currentChannel.type,
+        channelType: currentChannel && currentChannel.type,
```

Comment:
```suggestion
        channelType: currentChannel?.type,
```

what happens if its `undefined` do we have a default value?

Reply:
If that is undefined is because there is not current channel, so shouldn't be current stats (that should be a "transition" state). But anyway, I changed the code to be more "aware" of that.